build.sh

```diff
@@ -1,35 +1,91 @@
+  echo "              Default: \${DOCKER_REGISTRY}/\${DOCKER_ORG}/\${DOCKER_REPO}:\${TAG}"
+  echo "  DOCKER_SHORT_TAG The name of the short tag which is applied to the"
+  echo "              image. This is used to tag all patch releases to their"
+  echo "              containing version e.g. v2.5.1 -> v2.5"
+  echo "              Default: \${DOCKER_REGISTRY}/\${DOCKER_ORG}/\${DOCKER_REPO}:<MAJOR>.<MINOR>"
+  echo "  DOCKERFILE  The name of Dockerfile to use."
+  echo "              Default: Dockerfile"
+  echo "  DOCKER_FROM The base image to use."
+  echo "              Default: 'python:3.7-alpine'"
+  echo "  DOCKER_TARGET A specific target to build."
+  echo "              It's currently not possible to pass multiple targets."
+  echo "              Default: main"
+  echo "  HTTP_PROXY  The proxy to use for http requests."
+  echo "              Example: http://proxy.domain.tld:3128"
+  echo "              Default: undefined"
+  echo "  NO_PROXY    Comma-separated list of domain extensions proxy should not be used for."
+  echo "              Example: .domain1.tld,.domain2.tld"
+  echo "              Default: undefined"
   echo "  DEBUG       If defined, the script does not stop when certain checks are unsatisfied."
   echo "              Default: undefined"
   echo "  DRY_RUN     Prints all build statements instead of running them."
   echo "              Default: undefined"
+  echo "  GH_ACTION   If defined, special 'echo' statements are enabled that set the"
+  echo "              following environment variables in Github Actions:"
+  echo "              - FINAL_DOCKER_TAG: The final value of the DOCKER_TAG env variable"
+  echo "              Default: undefined"
   echo ""
   echo "Examples:"
   echo "  ${0} master"
   echo "              This will fetch the latest 'master' branch, build a Docker Image and tag it"
-  echo "              'peering-manager/peering-manager:latest'."
-  echo "  ${0} v1.2.0"
-  echo "              This will fetch the 'v1.2.0' tag, build a Docker Image and tag it"
-  echo "              'peeringmanager/peering-manager:v1.2.0' and 'peeringmanager/peering-manager:v1.2'."
-  echo ""
+  echo "              'respawner/peering-manager:latest'."
+  echo "  ${0} develop"
+  echo "              This will fetch the latest 'develop' branch, build a Docker Image and tag it"
+  echo "              'respawner/peering-manager:snapshot'."
+  echo "  ${0} v2.6.6"
+  echo "              This will fetch the 'v2.6.6' tag, build a Docker Image and tag it"
+  echo "              'respawner/peering-manager:v2.6.6' and 'respawner/peering-manager:v2.6'."
+  echo "  ${0} develop-2.7"
+  echo "              This will fetch the 'develop-2.7' branch, build a Docker Image and tag it"
+  echo "              'respawner/peering-manager:develop-2.7'."
+  echo "  SRC_ORG=respawner ${0} feature-x"
+  echo "              This will fetch the 'feature-x' branch from https://github.com/respawner/peering-manager.git,"
```

Comment:
Old repository URL used. Should now be `https://github.com/peering-manager/peering-manager.git`.

Reply:
Thanks for catching that, I thought I had gone through and changed those, but perhaps not. PR should be updated now.